platform-tests/src/test/java/org/junit/platform/commons/support/ModifierSupportTests.java

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright 2015-2018 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junit.platform.commons.support;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.lang.reflect.Member;
+
+import org.junit.jupiter.api.Test;
+import org.junit.platform.commons.util.PreconditionViolationException;
+
+/**
+ * Unit tests for {@link ModifierSupport}.
+ *
+ * @since 1.4
+ */
+class ModifierSupportTests {
+
```

Comment:
Please add tests for preconditions.

Reply:
Ahhhh... OK. I overlooked those,  because we typically test preconditions first and often in dedicated `preconditions()` test methods.